From 2b59e14a0023be9d084349d58ee156a49cc674bb Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@chromium.org>
Date: Wed, 15 Aug 2012 11:55:31 -0400
Subject: [PATCH] Parse out arm isa targets from dumpmachine

The current parsing logic of the dumpmachine tuple lacks any arm
cases which means tgt_isa never gets set, so for all arm targets,
we get detected as generic-gnu.  Add some basic arm checks here
so the automatic detection logic works.

Change-Id: Ie5e98142876025c6708604236bc519c0bdb09319
---
 build/make/configure.sh |    9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/build/make/configure.sh b/build/make/configure.sh
index 26eb864..eeb959a 100755
--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -593,6 +593,15 @@ process_common_toolchain() {
 
         # detect tgt_isa
         case "$gcctarget" in
+            armv6*)
+                tgt_isa=armv6
+                ;;
+            armv7*)
+                tgt_isa=armv7
+                ;;
+            armv5te*)
+                tgt_isa=armv5te
+                ;;
             *x86_64*|*amd64*)
                 tgt_isa=x86_64
                 ;;
-- 
1.7.9.7

